otl-normalizer/src/gsub.rs

```diff
@@ -148,6 +163,22 @@ impl LookupRules {
         all_rules.dedup();
         all_rules
     }
+
+    fn multisub_rules<'a>(&'a self, lookups: &[u16]) -> Vec<SingleRule<'a, MultiSubRule>> {
+        // normalization here is very simple: we're just going to do first-writer-wins
+        // on targets.
```

Comment:
Suggest elaborating on the implications of this. Are there inputs that will cause this approach problems? What are the advantages/disadvantages?